addons/mrp/views/mrp_workcenter_views.xml

```diff
@@ -32,10 +32,18 @@
             <field name="name">mrp.workcenter.tree</field>
             <field name="model">mrp.workcenter</field>
             <field name="arch" type="xml">
-                <tree string="Work Center">
+                <tree string="Work Center" multi_edit="1">
                     <field name="sequence" widget="handle"/>
-                    <field name="code"/>
                     <field name="name"/>
+                    <field name="code"/>
+                    <field name="alternative_workcenter_ids" widget="many2many_tags" optional="hide"/>
+                    <field name="productive_time" optional="hide"/>
+                    <field name="costs_hour"/>
+                    <field name="capacity"/>
+                    <field name="time_efficiency" optional="hide"/>
+                    <field name="oee_target" optional="hide"/>
+                    <field name="time_start" optional="hide" string="Setup Time"/>
```

Comment:
Should be change in the model instead to be consistent with the form view? (same for time_stop)